src/javascript/app/Stores/Modules/Contract/contract-store.js

```diff
@@ -183,6 +196,7 @@ export default class ContractStore extends BaseStore {
         this.is_ongoing_contract = false;
         this.sell_info           = {};
 
+        this.smart_chart.updateChartType('mountain');
```

Comment:
How come this is necessary?

Reply:
it was set before? i can remove it if it's not required anymore

Reply:
previously it set `this.chart_type` not the actual smart-chart chart type, I think the `this.chart_type` of contract-store can be removed since it doesn't seem to be used anywhere